### DISCUSSION TITLE
治安局提醒您：建议安装国家反诈中心

### DISCUSSION THREAD

**Discussion by su2569** [share121/inter-knot#1390] [其他]
[![image](https://bkimg.cdn.bcebos.com/pic/6a63f6246b600c3387448165dc1f460fd9f9d72a0b58?x-bce-process=image/format,f_auto/resize,m_lfit,limit_1,h_1000)](https://bkimg.cdn.bcebos.com/pic/6a63f6246b600c3387448165dc1f460fd9f9d72a0b58?x-bce-process=image/format,f_auto/resize,m_lfit,limit_1,h_1000)
https://www.coolapk.com/apk/com.hicorenational.antifraud
https://apps.apple.com/cn/app/%E5%9B%BD%E5%AE%B6%E5%8F%8D%E8%AF%88%E4%B8%AD%E5%BF%83/id1552823102?l=en-GB
https://sj.qq.com/appdetail/com.hicorenational.antifraud?fromcase=40002&from_wxz=1

**Comment by VacuolePaoo** [comment]
绳网是一个匿名论坛，为保护绳匠隐私，请不要引导绳匠进入外部网址，谢谢
